src/nginx/config.cc

```diff
@@ -312,6 +321,8 @@ esp_command_t esp_commands[] = {
     ESP_COMMAND("api_authentication", 1, 1, ESP_LOC_CONF,
                 esp_configure_api_authentication),
     ESP_COMMAND("server_config", 1, 1, ESP_LOC_CONF, esp_set_server_config),
+    // Set api base path
+    ESP_COMMAND("api_basepath", 1, 1, ESP_LOC_CONF, esp_set_api_basepath),
```

Comment:
I don't think we Nginx config for api_basepath.   Coming from start_esp argument, and server_config is good enough